projects/hockeypuck/services/hockeypuck/examples/basic.nix

```diff
@@ -0,0 +1,16 @@
+{ ... }:
+
+{
+  services.hockeypuck.enable = true;
```

Comment:
The service port needs to be open for it to be forwarded to the host. You can either do that directly here or [extend the NixOS module](https://github.com/ngi-nix/ngipkgs/blob/1de6b05dfd632cda7f39436c69cfb31b59d2a04f/projects/types.nix#L119-L140) and add an `openFirewall` option, then enable it.